extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/config/VertxConfiguration.java

```diff
@@ -58,6 +58,29 @@
     @ConfigItem(defaultValue = "20")
     public int internalBlockingPoolSize;
 
+    /**
+     * The queue size. For most applications this should be unbounded
+     */
+    @ConfigItem
+    public OptionalInt queueSize;
+
+    /**
+     * The executor growth resistance.
+     *
+     * A resistance factor applied after the core pool is full; values applied here will cause that fraction
+     * of submissions to create new threads when no idle thread is available. A value of {@code 0.0f} implies that
+     * threads beyond the core size should be created as aggressively as threads within it; a value of {@code 1.0f}
+     * implies that threads beyond the core size should never be created.
+     */
+    @ConfigItem
+    public float growthResistance;
```

Comment:
No default value here?

Reply:
I'd copied the properties across from our Thread configuration, which also didn't have defaults.

@dmlloyd should we include any defaults?

Reply:
The default should be `0.0`.  Since this configuration property is primitive, the config processor is set up to make primitive properties have a default of their normal initializer value (in this case `0.0`).

Reply:
So it's an implicit default and not one we need to add here?

Reply:
Right